Source/DafnyPipeline.Test/IntraMethodVerificationStability.cs

```diff
@@ -1,57 +1,164 @@
-using System;
 using System.Collections.Generic;
 using System.IO;
 using System.Linq;
 using Microsoft.Boogie;
 using Microsoft.Dafny;
 using Xunit;
+using Xunit.Abstractions;
 using BoogieProgram = Microsoft.Boogie.Program;
 using Parser = Microsoft.Dafny.Parser;
-using Type = System.Type;
 
 namespace DafnyPipeline.Test {
   // Main.Resolve has static shared state (TypeConstraint.ErrorsToBeReported for example)
   // so we can't execute tests that use it in parallel.
   [Collection("Singleton Test Collection - Resolution")]
   public class IntraMethodVerificationStability {
-    [Fact]
-    public void NoUniqueLinesWhenConcatenatingUnrelatedPrograms() {
-      var program = $@"
+    private readonly ITestOutputHelper testOutputHelper;
+
+    // All types of top level declarations.
+    // But also all types of AST nodes, they may all generate something using counters.
+    // Just trying to come up with this very large Dafny program is a hassle. Would be awesome if it could be generated based on the types.
+
+    readonly string originalProgram = $@"
```

Comment:
If interpolation (`$`) is not needed - which I think is true - can you please remove the `$` and replace all the double curly braces by single curly braces?
It'll become easier to read and also to copy in case we want to work on a variant of this test.